cassie-rails.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.homepage    = 'https://github.com/eprothro/cassie-rails'
   s.license     = 'MIT'
 
-  s.add_runtime_dependency 'cassie', '~> 1.0', ">= 1.0.4"
+  s.add_runtime_dependency 'cassie', '~> 1.1.0', ">= 1.1.1"
```

Comment:
this is overly strict on the pessimistic version

 s.add_runtime_dependency 'cassie', '~> Versions it should probably work with', ">= min version for it to work"

~> 1.1.0 means cannot work with 1.2, which not what we want (otherwise we have to update this every time we release cassie...